efficiency/bin/sdl_timing

```diff
@@ -114,6 +114,8 @@ if [ -z ${SPECIFICGPUVERSION} ] || [[ "${SPECIFICGPUVERSION}" == "explicit_cache
 fi
 
 echo "Total Timing Summary"
-echo "   Evt    Hits       MD       LS      T3       T5       pLS       pT5      pT3      TC       Event      Short           Rate"
-#echo "   Evt    Hits       MD       LS      T3       T5       pLS       pT5      pT3      TC       TCE      Event      Short           Loop      Effective"
+grep -h "Time for map " timing_temp.txt | cut -d " " -f 6 | awk '{ SUM += $1} END { print "Average time for map loading =",SUM/5,"ms" }' # 5 is the number of stream values run
+grep -h "Time for input " timing_temp.txt | cut -d " " -f 6 | awk '{ SUM += $1} END { print "Average time for input loading =",SUM/5,"ms" }' # 5 is the number of stream values run
+grep -h "Time for stream " timing_temp.txt | cut -d " " -f 6 | awk '{ SUM += $1} END { print "Average time for stream creation =",SUM/5,"ms"}' # 5 is the number of stream values run
```

Comment:
This time includes the event object creation time as well right? Although maybe the stream creation takes significantly longer so this is fine? Might be better to rename it, or even get rid of it since it only takes 0.1 ms.

Reply:
Nice catch, I agree it should be over 21. Yea sdl::event creation sounds more clear to me as well.

Reply:
Fixed in [6e2b1ba](https://github.com/SegmentLinking/TrackLooper/pull/305/commits/6e2b1baf442836cf1b857b4899777a49820d01b5).